thruster/examples/hello_world.rs

```diff
@@ -11,13 +11,14 @@ async fn plaintext(mut context: Ctx, _next: MiddlewareNext<Ctx>) -> Ctx {
     context
 }
 
-fn main() {
+#[tokio::main]
```

Comment:
Let's keep this and the most basic hyper with the `start` syntax, as these should be as simple + easy as possible.